SECURITY.md

```diff
@@ -6,9 +6,9 @@ The versions of the project that are currently supported with security updates.
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 4.x | :white_check_mark: |
-| 3.x | :white_check_mark: |
-| < 3.0 | :x: |
+| 4.x     | :white_check_mark: |
+| 3.x     | :white_check_mark: |
+| < 3.0   | :x:                |
```

Comment:
> I wonder if it can invalidate the existing Crowdin translations.

Any change invalidates.

>  It could be challenging to review.

What is challenging?

Reply:
> Any change invalidates.

I hope Crowdin trims the table cells before considering it as a source of the translations. It might be OK. 

> What is challenging?

The git diff would impact more lines that it needs to, but the visual diff would be left unchanged.

In any case, the downside seems much less important than the upside of using Prettier. I don't think that it should stop us form moving forward :+1:.